packages/topkit-analytics-charts/src/CategoriesChart/types.ts

```diff
@@ -0,0 +1,22 @@
+type Value = {
+  id: string
+  value: number
+}
+
+export type Bar = {
+  id: string
+  values: [
+    {
+      id: string
+      values: Value[]
+    },
+    {
+      id: string
+      values: Value[]
+    }
+  ]
+}
+
+export type Labels = Record<string, string>
```

Comment:
nit: `Labels` variable name\ makes me think that it's an array type.

```suggestion
export type LabelMap = Record<string, string>
```